dsl/dsl.cc

```diff
@@ -93,6 +94,13 @@ class DSLReplacer {
                         replaceNodes[stat.get()] = std::move(nodes);
                         return;
                     }
+
+                    // This one is also a little different: it gets the classDef
+                    nodes = Mattr::replaceDSL(ctx, send, classDef.get());
+                    if (!nodes.empty()) {
+                        replaceNodes[stat.get()] = std::move(nodes);
+                        return;
+                    }
```

Comment:
I'm a little reluctant to have these continually diverge from each other. If it's dependent on the classDef, why not just use `patchDSL` like `Command` / `Rails` / `OpusEnum` / `Prop` which also need to access the classDef?

Reply:
Or at the very least, can we make it take only the `classDef->kind` instead of the entire `ClassDef`? I just don't want people lurking things into the ClassDef via the replaceDSL call.

Reply:
passed classDef->kind instead.